style.css

```diff
@@ -29,6 +30,10 @@ a {
   text-decoration: none;
 }
 
-li {
+.list {
```

Comment:
this utility can be more specific as it may be confusing.... for e.g. `.list-st-none` or `.list-none` sounds more appropriate